modules/claims_api/spec/fixtures/form_526_evss.json

```diff
@@ -43,13 +43,21 @@
         ]
       }
     ],
+    "homelessness": {
+      "pointOfContact": {
+        "pointOfContactName": "John Doe",
+        "primaryPhone": "555-555-5555"
+      },
+      "currentlyHomeless": {
+        "homelessSituationType": "FLEEING_CURRENT_RESIDENCE",
+        "otherLivingSituation": "other living situations"
+      }
+    },
```

Comment:
the "homelessness" object needs to be moved under the "veteran" top-level object. that's why it looks like it's not obeying the `"required": [ "pointOfContact" ],` constraint

Reply:
Fixed